uninstall.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+rm -f ~/.local/bin/trash
+rm -f ~/.local/bin/trash-empty
+rm -f ~/.local/bin/trash-list
+rm -f ~/.local/bin/trash-put
+rm -f ~/.local/bin/trash-restore
+rm -f ~/.local/bin/trash-rm
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/cmds.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/cmds.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/empty.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/empty.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/fs.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/fs.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/fstab.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/fstab.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/__init__.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/__init__.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/list_mount_points.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/list_mount_points.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/list.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/list.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/put.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/put.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/restore.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/restore.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/rm.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/rm.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/trash.py
+rm -f ~/.local/lib/python2.7/site-packages/trashcli/trash.pyc
+rm -f ~/.local/lib/python2.7/site-packages/trash_cli-0.17.1.14.egg-info
```

Comment:
Thank you for your submission.
These paths are specific for your installation, you can use `pip uninstall trash-cli` to unistall trash-cli